tests/test_deserialize_json.py

```diff
@@ -39,7 +40,7 @@ class TestDeserializeJson(TestCase, SnapshotMixin, DeepCompareMixin):
     @patch('cyclonedx.model.bom_ref.uuid4', side_effect=uuid_generator(0, version=4))
     def test_prepared(self, get_bom: Callable[[], Bom], *_: Any, **__: Any) -> None:
         # only latest schema will have all data populated in serialized form
-        snapshot_name = mksname(get_bom, SchemaVersion.V1_4, OutputFormat.JSON)
+        snapshot_name = mksname(get_bom, LATEST_SUPPORTED_SCHEMA_VERSION, OutputFormat.JSON)
```

Comment:
:exclamation: TODO for myself:
need to think about an optimization here. maybe change the name to `...latest`, instead of having the version encoded in the name.

Reply:
caused #491